.github/workflows/pkgdown.yaml

```diff
@@ -0,0 +1,64 @@
+on:
+  push:
+    branches: pkgdown
+
+name: pkgdown
+
+jobs:
+  pkgdown:
+    runs-on: ubuntu-18.04
+    env:
+      RSPM: https://packagemanager.rstudio.com/cran/__linux__/bionic/latest
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v1
+        id: install-r
+
+      - uses: r-lib/actions/setup-pandoc@v1
+
+      - name: Install pak and query dependencies
+        run: |
+          install.packages("pak", repos = "https://r-lib.github.io/p/pak/dev/")
+          saveRDS(pak::pkg_deps("local::.", dependencies = TRUE), ".github/r-depends.rds")
+        shell: Rscript {0}
+
+      - name: Cache R packages
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.R_LIBS_USER }}
+          key: ${{ runner.os }}-${{ steps.install-r.outputs.installed-r-version }}-1-${{ hashFiles('.github/r-depends.rds') }}
+          restore-keys: ${{ runner.os }}-${{ steps.install-r.outputs.installed-r-version }}-1-
+
+      - name: Install system dependencies
+        if: runner.os == 'Linux'
+        run: |
+          pak::local_system_requirements(execute = TRUE)
+          pak::pkg_system_requirements("pkgdown", execute = TRUE)
+        shell: Rscript {0}
+
+      - name: Install package and dependencies
+        run: |
+          pak::local_install_dev_deps(upgrade = TRUE)
+          pak::pkg_install("r-lib/pkgdown")
+          pak::pkg_install("rstudio/quillt")
+        shell: Rscript {0}
+
+      - name: Install package
+        run: R CMD INSTALL .
+
+      - name: Build pkgdown site
+        run: pkgdown::build_site(new_process = FALSE)
+        shell: Rscript {0}
+
+      - name: Deploy to Netlify
+        # only deploy on the fork for now
+        if: github.repository_owner == 'apreshill'
+        # NETLIFY_AUTH_TOKEN added in the repo's secrets
+        env:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+        uses: netlify/actions/cli@master
+        with:
+          args: deploy --prod --dir reference
```

Comment:
Does this push to https://flexdashboard-pkg.netlify.app? How does the redirect to https://pkgs.rstudio.com/ occur (would be great to have a comment about that here)

Reply:
> How does the redirect to https://pkgs.rstudio.com/ work (would be great to have a comment about that here)?

It has to be done in pkgs.rstudio.com/ website directly - I'll add a redirect rule when this website is deployed.

> Does this push to https://flexdashboard-pkg.netlify.app?

Did you already set the secrets in this repository ? 
Currently they are set in @apreshill fork hence why the deployment happens there.

Reply:
> Did you already set the secrets in this repository ?

No, I'm not sure I have (or should have) the proper secrets either. I just granted @apreshill admin rights, so perhaps she can add them.

Reply:
Thanks! Will do. This vignette looks great! OK if I push some suggestions to this branch?

Reply:
Thanks, by all means!

Reply:
Hey Carson- I left a few HTML comments in there for you if you could take a look. Also added some `##` headers to populate the TOC on the right to help with skimming.